deploy/buildtools/buildfarm.py

```diff
@@ -69,7 +69,7 @@ def __init__(self, args: Dict[str, Any]) -> None:
     @staticmethod
     @abc.abstractmethod
     def NAME() -> str:
```

Comment:
https://github.com/firesim/firesim/pull/1006#discussion_r842194562

Reply:
Decision offline was to remove this and just have the class names be specific and clear.